src/common/envalid.ts

```diff
@@ -1,6 +1,21 @@
-import { config } from 'dotenv'
+import { parse } from 'dotenv'
 import { bool, cleanEnv, json, str } from 'envalid'
-import { existsSync } from 'fs'
+import { existsSync, readFileSync } from 'fs'
+
+export const dotEnvParse = (path: string): void => {
```

Comment:
I thought that dotEnvParse should be part of different PR right ?

Reply:
Whoops, branched from wrong branch, thanks for helping me fix it :)